pkg/kubelet/server/server.go

```diff
@@ -155,6 +156,9 @@ type HostInterface interface {
 	ResyncInterval() time.Duration
 	GetHostname() string
 	LatestLoopEntryTime() time.Time
+
+	// Used by fs_resource_analyzer
```

Comment:
+1. Go also requires these comments to be describing the method's functionality.